### PR TITLE
Feature/milestone 3

### PR DESCRIPTION
**Pull request by kunmi02**
On this method we implented actions for the add and remove button

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
 const Book = (props) => {
-  const { book } = props;
+  const { book, handleChange } = props;
   return (
     <tr>
       <td>{book.id}</td>
       <td>{book.title}</td>
       <td>{book.category}</td>
+      <td><button type="button" onClick={(event) => handleChange(event, book)}>Remove Book</button></td>
```
Comment (bellom) [inline src/components/Book.js]: This is a button to remove a book, you should pass a function to remove a book example `handleRemoveBook` which you already have in `BookList component`

File: src/components/Book.js [hunk 2]
```diff
@@ -1,18 +1,20 @@
 import PropTypes from 'prop-types';
 
 const Book = (props) => {
-  const { book } = props;
+  const { book, handleChange } = props;
```
Comment (bellom) [inline src/components/Book.js]: This prop should be a remove function `handleRemoveBook `

Screenshot from pathwright:
image

File: src/containers/BooksForm.js [hunk 3]
```diff
@@ -57,4 +82,9 @@ class BooksForm extends React.PureComponent {
   }
 }
 
-export default BooksForm;
+BooksForm.propTypes = {
+  createBook: PropTypes.func.isRequired,
+};
+
+// export default BooksForm;
```
Comment (bellom) [inline src/containers/BooksForm.js]: kindly delete commented cod, you could use git history instead.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Review by oloomoses — APPROVED** [review]
Hi Team, 

Your milestone is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

------

Slack: Oloo Moses

**Pull request merged**